api/src/org/apache/cloudstack/api/command/admin/volume/ListVolumesCmdByAdmin.java

```diff
@@ -34,32 +34,5 @@
 @APICommand(name = "listVolumes", description = "Lists all volumes.", responseObject = VolumeResponse.class, responseView = ResponseView.Full, entityType = {Volume.class},
         requestHasSensitiveInfo = false, responseHasSensitiveInfo = false)
 public class ListVolumesCmdByAdmin extends ListVolumesCmd {
-    public static final Logger s_logger = Logger.getLogger(ListVolumesCmdByAdmin.class.getName());
-
-    @Parameter(name=ApiConstants.POD_ID, type=CommandType.UUID, entityType=PodResponse.class,
-            description="the pod id the disk volume belongs to")
-    private Long podId;
```

Comment:
@rafaelweingartner Is podId an unused parameter?

Reply:
It is in the superclass as well. These parameters were declared twice. One in the "admin" class, and other in the "normal" one.

Reply:
make sense to remove if parent class has them already declared.